plugins/metadata/lib/model/IMetadataPeer.php

```diff
@@ -9,4 +9,6 @@
 	public static function retrieveByPK($pk, PropelPDO $con = null);
 	
 	public static function validateMetadataObjects($profileField, $objectIds, &$errorMessage);
+
+	public static function getEntry($objectId);
```

Comment:
1. all other functions in this interface are static
2. no need to access any data members to do this function